langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -106,6 +126,11 @@ public isolated function ensureType(any|error v, typedesc<any> t = <>) returns t
 #
 # Returns true if read-only, false otherwise.
 #
+# ```ballerina
+# int[] & readonly finalScores = [21, 12, 33, 45, 81];
```

Comment:
shall we rename the variable to `scores`. it will be short and more appropriate for documentation. wdyt?